run.ps1

```diff
@@ -52,8 +52,8 @@ in the file are overridden by command line parameters.
 Example config file:
 ```json
 {
-  "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
-  "channel": "dev",
+  "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/master/tools/korebuild.schema.json",
+  "channel": "master",
```

Comment:
Is this intentional? For the 2.1 branch to point to `master`?

Reply:
This part of the file is just docs. korebuild.json in this repo still points to the release/2.1 channel.